frontend/src/features/permits/apiManager/endpoints/endpoints.ts

```diff
@@ -1,24 +1,37 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import { VEHICLES_URL } from "../../../../common/apiManager/endpoints/endpoints";
 
-const PERMITS_API_BASE = `${VEHICLES_URL}/permits`;
-const APPLICATIONS_API_BASE = `${PERMITS_API_BASE}/applications`;
+const APPLICATIONS_API_BASE_2 = (companyId: string) =>
+  `${VEHICLES_URL}/companies/${companyId}/applications`;
+
+const APPLICATIONS_API_BASE_22 = (_s: any, companyId: string) => {
```

Comment:
Hmm... what's with the _2 and _22 naming?

Reply:
I added this as  temporary replacement to deal with compilation issues. Will fix this.